src/main/java/com/rabbitmq/client/ChannelRpcTimeoutException.java

```diff
@@ -0,0 +1,35 @@
+package com.rabbitmq.client;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Exception thrown when a channel times out on a RPC call.
+ * @since 4.1.0
+ */
+public class ChannelRpcTimeoutException extends IOException {
```

Comment:
Please rename this to `ChannelContinuation…`. `RPC` will mislead those who use higher level libraries that may claim to support "RPC".